ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -3,92 +3,75 @@ vlan_interfaces:
 {% for tenant in network_services_data.tenants %}
 {%     for vrf in tenant.vrfs %}
 {%         for svi in vrf.svis %}
-{# Detect if a svi_profile exists #}
-{# If exists, create a shortpath to access profile data #}
-{%             if svi.profile is arista.avd.defined %}
-{%                 set svi_profile = svi_profiles | arista.avd.default([]) |
-                                                                      arista.avd.convert_dicts('profile') |
-                                                                      selectattr('profile', 'arista.avd.defined', svi.profile) |
-                                                                      first %}
-{%             endif %}
-{%             if svi_profile.parent_profile is arista.avd.defined %}
-{%                 set parent_profile = svi_profiles | arista.avd.default([]) |
-                                                                      arista.avd.convert_dicts('profile') |
-                                                                      selectattr('profile', 'arista.avd.defined', svi_profile.parent_profile) |
-                                                                      first %}
-{%             endif %}
-{%             set svi_settings = {} | ansible.builtin.combine(parent_profile | arista.avd.default({}),
-                                               svi_profile | arista.avd.default({}),
-                                               svi, recursive=true, list_merge='replace') %}
 {#             Structured config should not be merged recursively, but will be taken directly from the most specific level #}
 {%             set svi_structured_config = svi.nodes[inventory_hostname].structured_config | arista.avd.default(
```

Comment:
see other suggestion on moving this to logic. (Also this would not work since you are no longer setting svi_profile and parent_profile)